### PR TITLE
ColorPicker => MultiColorPicker

### PR DESCRIPTION
**Pull request by VeraZab**

### CODE REVIEW

**Comment by nicolaskruchten** [comment]
Behaviour looks good, 💃 from me subject to @dmt0 's code review

File: src/components/widgets/ColorPicker.js [hunk 1]
```diff
@@ -152,7 +152,7 @@ class ColorPicker extends Component {
 
 ColorPicker.propTypes = {
   onColorChange: PropTypes.func.isRequired,
-  selectedColor: PropTypes.string,
+  selectedColor: PropTypes.any,
```
[the remark this answers is not in the document]
Comment (dmt0) [reply src/components/widgets/ColorPicker.js]: There's a handy construct for this:
`PropTypes.oneOfType([PropTypes.string, PropTypes.number])`

File: src/components/fields/MultiColorPicker.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+        props.fullValue.every(v => v[1] === props.fullValue[0][1])
+          ? 'single'
+          : 'multiple',
+    };
+    this.setColor = this.setColor.bind(this);
+    this.setColors = this.setColors.bind(this);
+  }
+
+  setColor(color) {
+    this.props.updatePlot(color);
```
Comment (dmt0) [inline src/components/fields/MultiColorPicker.js]: why not just pass updatePlot as a prop?

File: src/components/fields/MultiColorPicker.js [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+  }
+
+  render() {
+    const _ = this.context.localize;
+    const constantOptions = [
+      {label: _('Single'), value: 'single'},
+      {label: _('Multiple'), value: 'multiple'},
+    ];
+    const selectedConstantColorOption = this.props.parentState
+      ? this.props.parentState.selectedConstantColorOption
```
Comment (dmt0) [inline src/components/fields/MultiColorPicker.js]: parentState only has one variable, why not just have parentSelectedConstantColorOptions?

Comment (VeraZab) [reply src/components/fields/MultiColorPicker.js]: I'm using parent state to pass MarkerColor state, it has more than one variable.

File: src/components/fields/MultiColorPicker.js (hunk 2 above)
Comment (VeraZab) [reply src/components/fields/MultiColorPicker.js]: It is passed, from the connect wrapper, I just prefer having a separate function, 
one setColor, one setColors, that's all

File: src/components/fields/MultiColorPicker.js (hunk 3 above)
Comment (dmt0) [reply src/components/fields/MultiColorPicker.js]: I'm not seeing other variables

**Pull request merged**